CMakeLists.txt

```diff
@@ -54,6 +55,10 @@ if(WIN32 OR APPLE)
 endif()
 option(ENABLE_FILE_RESOURCE "Enable File Resource" On)
 
+if(WIN32)
```

Comment:
I don't think we need to set this off here - it will print a warning if someone tries to turn it on. We just need to make sure it's turned off for the Windows CI.